.github/workflows/publish_docker_images.yml

```diff
@@ -7,6 +7,7 @@ on:
     tags:
       - "v*"
       - "happy-plazza"
+      - "guilload"
```

Comment:
maybe we can put something like dev?

can be useful to test some image on kube.

Reply:
I added `qw*`.